src/serialization/base64.cpp

```diff
@@ -0,0 +1,90 @@
+#include "serialization/base64.hpp"
+
+#include <vector>
+
+namespace {
+const std::vector<int>& decode_map(){
+	static std::vector<int> T(256, -1);
+	if(T['A'] == -1){
+		for(int i=0; i<64; i++) {
+			T["ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/"[i]] = i;
+		}
+	}
+	return T;
+}
+const std::string itoa64_map = "./0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
+char itoa64(unsigned value){
+	return itoa64_map[value & 0x3f];
+}
+}
+
+namespace base64 {
+std::string decode(utils::string_view in)
+{
+	const std::vector<int>& T = decode_map();
```

Comment:
Please don't use an uppercase variable name. Uppercase identifiers usually mean classes. And T in particular is usually a template parameter by convention.